ibm/mas_devops/playbooks/mas/install-suite.yml

```diff
@@ -36,7 +36,9 @@
     # Domain prefix is whatever you want to append to your DNS entry to make it unique
     cis_subdomain: "{{ lookup('env', 'CIS_SUBDOMAIN') }}"
     # generate your apikey in IBM Cloud
-    ibmcloud_apikey: "{{ lookup('env', 'IBMCLOUD_APIKEY') }}"
+    # ibmcloud_apikey: "{{ lookup('env', 'IBMCLOUD_APIKEY') }}"
+    # generate the cis_apikey for a Service ID in IBM Cloud with an 'Access Policy' of Editor/Manager
+    cis_apikey: "{{ lookup('env', 'CIS_APIKEY') }}"
```

Comment:
let's add a defaut value in case CIS_APIKEY is not set? so we do not break things that are currently working... let's add a `| default` in this lookup, and use IBMCLOUD_APIKEY if CIS one is not set (there are examples to set the default in some playbooks)

Reply:
Added the default to be the IBMCLOUD_APIKEY which is also performing a **lookup** for the environment variable.